content/guides/output-management.md

```diff
@@ -221,6 +222,20 @@ __webpack.config.js__
   };
 ```
 
+__src/index.html__
```

Comment:
It's referenced here: "You can also take a look at html-webpack-template which provides a couple of extra features in addition to the default template." 😉